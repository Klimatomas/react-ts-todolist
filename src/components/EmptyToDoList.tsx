import React = require("react");
import { SFC } from "react";

interface IEmptyToDoListProps {
  createToDo: () => void;
}

const EmptyToDoList: SFC<IEmptyToDoListProps> = (
  props: IEmptyToDoListProps
) => (
  <div id="emptyToDo">
    <h4 id="emptyToDoText">
      Nothing in the list! Click the button below to get started
    </h4>
    {/* todo add glyphicon / styling (font awesome?) */}
  </div>
);

export default EmptyToDoList;
