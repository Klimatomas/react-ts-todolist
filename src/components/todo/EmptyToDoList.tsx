import React = require("react");
import { SFC } from "react";

const EmptyToDoList: SFC<{}> = () => (
  <div id="emptyToDo">
    <h4 id="emptyToDoText">
      Nothing in the list! Click the button below to get started
      <div>asd</div>
    </h4>
  </div>
);

export default EmptyToDoList;
