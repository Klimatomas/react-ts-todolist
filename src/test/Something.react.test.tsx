import React from "react";
import renderer from "react-test-renderer";
import EmptyToDoList from "../components/EmptyToDoList";

test("Empty todo list gets rendered", () => {
  const component = renderer.create(<EmptyToDoList />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test ("kokot sa stane", () => {
  const a = 1;
  const b = 2;
  const val = a+b;
  expect(val).toEqual(3);
})