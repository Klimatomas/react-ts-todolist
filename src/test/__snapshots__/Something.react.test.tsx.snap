// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty todo list gets rendered 1`] = `
<div
  id="emptyToDo"
>
  <h4
    id="emptyToDoText"
  >
    Nothing in the list! Click the button below to get started
  </h4>
</div>
`;
